assets/js/stripe-payment-request.js

```diff
@@ -609,9 +609,21 @@ jQuery( function( $ ) {
 					paymentRequest.show();
 				}
 			});
+			
+			$( document.body ).on( 'wc_stripe_unblock_payment_request_button wc_stripe_enable_payment_request_button', function () {
+				wc_stripe_payment_request.unblockPaymentRequestButton( prButton );
+			} );
+			
+			$( document.body ).on( 'wc_stripe_block_payment_request_button', function () {
+				wc_stripe_payment_request.blockPaymentRequestButton( prButton, 'wc_request_button_is_blocked' );
+			} );
+
+			$( document.body ).on( 'wc_stripe_disable_payment_request_button', function () {
+				wc_stripe_payment_request.blockPaymentRequestButton( prButton, 'wc_request_button_is_disabled' );
+			} );
```

Comment:
Is the `wc_stripe_disable_payment_request_button` event ever triggered? As far as I can tell this is new and the event isn't triggered 🤔

Reply:
Not on this PR. The handler is being added so plugins can take advanced of it to programmatically disable the button. That's what was proposed on the [original issue](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1509)

Reply:
Ah, makes sense! Thank you!